develop/tutorials/articles/110-portlets/00-portlets-intro.markdown

```diff
@@ -64,6 +65,8 @@ flexible than servlets. Each portlet phase executes different operations:
   lifecycle. This lets a portlet serve dynamic content without running the 
   Render phase on all portlets on a page. The Resource-serving phase handles 
   AJAX requests. 
+- **Header:** Loads resource dependencies, such as CSS, prior to the Render 
```

Comment:
@jhinkey Recommend "Provides the ability for the developer to specify resource dependencies" since it is the job of the browser to actually "load" them.